ci/is_release_tag.bash

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+set -e
+
+tag=$1
+
+git clone https://github.com/snowplow/iglu-central.git tmp/iglu-central
```

Comment:
Can we add some detailed comments here explaining that this is a workaround for not fetched git ref.